xonsh/prompt/base.py

```diff
@@ -22,13 +23,61 @@
 from xonsh.prompt.vc import current_branch, branch_color, branch_bg_color
 from xonsh.prompt.gitstatus import gitstatus_prompt
 from xonsh.prompt.times import _localtime
+import logging
```

Comment:
standard library imports should be grouped with other standard library imports at the top.

Also, standard xonsh doesn't otherwise use logging.  Is there any way that you use print(..., file=sys.stderr), etc so that we don't incur the import time for this module?

Reply:
From my very basic test, it seems like logging takes 0.53 ms to import

```
base scopatz@artemis ~ $ timeit! python -c "pass"
100 loops, best of 3: 4.67 ms per loop
base scopatz@artemis ~ $ timeit! python -c "import logging"
100 loops, best of 3: 5.2 ms per loop
```

Reply:
Ok replaced it with print statement